api/src/opentrons/protocol_engine/commands/absorbance_reader/initialize.py

```diff
@@ -70,7 +70,10 @@ async def execute(self, params: InitializeParams) -> SuccessData[InitializeResul
                 supported_wavelengths
             )
             if unsupported_wavelengths:
-                raise ValueError(f"Unsupported wavelengths: {unsupported_wavelengths}")
+                raise ValueError(
+                    f"Unsupported wavelengths: {unsupported_wavelengths}, "
+                    f" use {supported_wavelengths} instead."
```

Comment:
Small change to make this a bit more like the error we have for loading an invalid module, which I like the format of:

```suggestion
                    f"Unsupported wavelengths: {unsupported_wavelengths}. "
                    f" Use one of {supported_wavelengths} instead."
```